ci-operator/step-registry/operator/install/operator-install-commands.sh

```diff
@@ -47,3 +47,12 @@ if [ "${INSTALL_FROM_IIB}" = "true" ]; then
 fi
 
 ${RUN_COMMAND}
+
+for operator_value in $(env | grep -E '^OPERATOR[0-9]+_CONFIG' | sort --version-sort); do
```

Comment:
please save `$(env | grep -E '^OPERATOR[0-9]+_CONFIG' | sort --version-sort)` and re-use